tests/Behat/Page/Admin/Channel/UpdatePage.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\InvoicingPlugin\Behat\Page\Admin\Channel;
+
+use Sylius\Behat\Page\Admin\Channel\UpdatePage as BaseUpdatePage;
+
+final class UpdatePage extends BaseUpdatePage implements UpdatePageInterface
```

Comment:
Since, as far as I know, all conditions in this plugin are written using `yoda notation`, WDYT about replacing

```
$this->getElement('company')->getValue() === $company;
```

with 

```
$company === $this->getElement('company')->getValue();
```

and do the same with another examples?

Reply:
Fixed 👍